string classification
---
bool success
string message
